.github/workflows/nightly.yml

```diff
@@ -0,0 +1,40 @@
+name: Nightly Checks
+
+on:
+  schedule:
+    # Every night at midnight
+    - cron:  '0 0 * * *'
+
+jobs:
+  # This won't build on GA environment because of missing TSS
+  # libraries. To be reactivated later or to pass using Docker.
+  dependencies:
+    name: Check for unused dependencies
+    if: false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install latest Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+      - name: Fetch submodules
+        run: git submodule update --init
+      - name: Install cargo udeps
+        run: cargo install cargo-udeps --locked
+      - name: Execute cargo udeps
+        run: cargo +nightly udeps
```

Comment:
I just noticed - do you need the `+nightly` if you specify `with: toolchain: nightly` above?

Reply:
Hmm yeah probably not, I just copy-pasted the command to make sure it worked 🤠